spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -435,6 +435,49 @@ def f(x):
         assert content['found']
 
 
+@flaky(max_runs=3)
+@pytest.mark.skipif(not PY3,
+                    reason="Only meant for Python 3")
+def test_dask_multiprocessing(tmpdir):
+    """
+    Test that dask multiprocessing works on Python 3.
+    """
+    # Command to start the kernel
+    cmd = "from spyder_kernels.console import start; start.main()"
+
+    with setup_kernel(cmd) as client:
+        # Remove all variables
+        client.execute("%reset -f")
+        client.get_shell_msg(block=True, timeout=TIMEOUT)
+
+        # Write multiprocessing code to a file
+        # Runs two times to verify that in the second case it doesn't break
+        code = """
+from dask.distributed import Client
+
+if __name__=='__main__':
+    client = Client()
+    client.close()
+    x = 'hello'
+
+    client = Client()
+    client.close()
+    x = 'hello'
```

Comment:
Please remove these lines and instead run the file twice with `runfile` to exactly reproduce the conditions specified by the user on the Spyder issue.